node/tests/financials_test.rs

```diff
@@ -8,23 +8,27 @@ use masq_lib::messages::{
     UiShutdownRequest, NODE_UI_PROTOCOL,
 };
 use masq_lib::test_utils::ui_connection::UiConnection;
-use masq_lib::test_utils::utils::ensure_node_home_directory_exists;
+use masq_lib::test_utils::utils::{ensure_node_home_directory_exists, open_all_file_permissions};
 use masq_lib::utils::find_free_port;
 use node_lib::accountant::db_access_objects::payable_dao::{PayableDao, PayableDaoReal};
 use node_lib::accountant::db_access_objects::receivable_dao::{ReceivableDao, ReceivableDaoReal};
 use node_lib::accountant::db_access_objects::utils::{from_time_t, to_time_t};
 use node_lib::accountant::gwei_to_wei;
 use node_lib::test_utils::make_wallet;
 use std::time::SystemTime;
+use masq_lib::constants::POLYGON_MAINNET_FULL_IDENTIFIER;
+use node_lib::database::db_initializer::{DbInitializationConfig, DbInitializer, DbInitializerReal};
+use node_lib::db_config::config_dao::{ConfigDao, ConfigDaoReal};
 use utils::MASQNode;
 
 #[test]
-fn financials_command_retrieves_payable_and_receivable_records() {
+fn financials_command_retrieves_payable_and_receivable_records_integration() {
```

Comment:
Because this test wasn't named `xxx_integration`, it wasn't being run. Since it wasn't being run, it didn't work. We fixed it.